handbook/engineering/roles.md

```diff
@@ -15,6 +15,28 @@ Sofware Engineers are individual contributors who build software to satisfy the
 - Help your teammates by empathetically [reviewing code](code_reviews.md) and [RFCs](../communication/rfcs/index.md).
 - Interview candidates and suggest improvements to our hiring process.
 
+## Project Lead
+
+Project Leads are Software Engineers that lead a specific project team to achieve its goals. This role is an additional set of responsibilities that any engineer may take on once they demonstrate the necessary skills.
+
+While Project Leads need to have strong technical skills, they are not necessarily the most technically experienced engineer on the team. People and organizational skills are as important as the technical chops, so that Project Leads can effectively coordinate and align with others.
```

Comment:
Nit: I just find "chops" to clash a bit in terms of register 🙂 

```suggestion
While Project Leads need to have strong technical skills, they are not necessarily the most technically experienced engineer on the team. People and organizational skills are as important as technical know-how, so that Project Leads can effectively coordinate and align with others.
```